model_analyzer/record/types/perf_latency_p99.py

```diff
@@ -18,13 +18,12 @@
 
 
 @total_ordering
-class PerfLatency(Record):
+class PerfLatencyP99(Record):
```

Comment:
I think we would save a lot of code, if we still kept a metric titled "perf_latency" which we document as deprecated AND as equivalent to p99 latency. We can remove it later on, instead of adding code to convert back and forth

Reply:
@Tabrizian what do you think of just keeping both `perf_latency` and `perf_latency_99` and just having them parse and refer to the same metric? This will remove the necessity to convert one to the other. If the use of `perf_latency` is detected by perf_analyzer.py we can just log a warning.

Reply:
I agree.  I tried to suggest the same in this [comment](https://github.com/triton-inference-server/model_analyzer/pull/187#discussion_r688104318).

Reply:
> Can we use autofill function and not remove the `perf_latency` from the list of configs? If the user has provided `perf_latency` we will set the same value for `perf_latency_p99` and if the user has provided a value for `perf_latency_p99` we set it for `perf_latency`.

I see what you mean by using the autofill function. It is adjusted to match behaviors between `perf_latency` and `perf_latency_p99` in analyze constraints.